### ISSUE TITLE
support property methods with no accessor prefix

### ISSUE THREAD

**Issue by machow** [machow/siuba#198]
Methods like `dtype` are accessed as properties (or are attributes in some cases):

```
from siuba.data import mtcars

mtcars.dtype
mtcars.hp.dtype
```

However, CallTreeLocal currently is only configured to assume something is a function call if it follows a prefix (e.g. `str`). This is a problem if you are trying to transform a word like `dtype`.

However, consider these cases...

```
mtcars.hp.dtype
mtcars.hp.values.dtype
mtcars.hp.dtype.upper()

g_cyl = mtcars.groupby('cyl')
g_cyl.hp.dtype.str.upper()
```

It seems like the two keys here are

1. no method should allow returning anything other than a grouped Series
2. each stage in the operation should validate that (since it runs in the "outer" part, not once per group)
3. assuming every method returns a grouped Series, we should be able to specify which top-level attributes are also "calls"

**Comment by machow** [comment]
Will be resolved by #308

**Comment by machow** [comment]
I don't know if I accidentally fixed this twice, but resolved by #245, v0.0.25
